javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,133 @@
+// WARM UP TASK
+const names = [
+  'Peter',
+  'Ahmad',
+  'Yana',
+  'kristina',
+  'Rasmus',
+  'Samuel',
+  'katrine',
+  'Tala',
+];
+
+const nameToRemove = 'Ahmad';
+
+const indexToRemove = names.includes(nameToRemove);
+if (indexToRemove !== 1) {
+  names.splice(indexToRemove, 1);
+}
+
+console.log(names);
+
+//TASK1 : When will we be there??
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function calculateTravelTime(info) {
+  const totalHours = info.destinationDistance / info.speed;
+  const hours = Math.floor(totalHours);
+  const minutes = Math.round((totalHours - hours) * 60);
+  return `${hours} hours and ${minutes} minutes`;
```

Comment:
Excellent variable naming! ⭐ 

To be a bit annoying: if something took exactly one minute, you would return "0 hours and 1 minutes"; can you think of a way to improve this? 😄